controllers/sriov-controller/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine
```

Comment:
If you want this built from the top-level, please add the appropriate bits into `Makefile` and `.nsm.mk` and I'll review those. Ping me if you have questions, but it should be straightforward. Also, to publish to Docker Hub we'll need to first create the repository on that side. Once this commit merges (if it had those changes), it would then publish to Docker Hub.